elide-core/src/main/java/com/yahoo/elide/core/SecurityMode.java

```diff
@@ -10,5 +10,6 @@
  */
 public enum SecurityMode {
     BYPASS_SECURITY,
+    VERBOSE_ERRORS,
```

Comment:
maybe `ACTIVE_VERBOSE` or `ACTIVE_WITH_ERRORS`? We should try to maintain some consistency so we don't have any confusion on what security modes are doing.